connector/kinesis-asl-assembly/pom.xml

```diff
@@ -59,16 +59,6 @@
       <artifactId>commons-lang</artifactId>
       <scope>provided</scope>
     </dependency>
-    <dependency>
-      <groupId>com.google.protobuf</groupId>
-      <artifactId>protobuf-java</artifactId>
-      <version>2.6.1</version>
-      <!-- 
-         We are being explicit about version here and overriding the 
-         spark default of 2.5.0 because KCL appears to have introduced 
-         a dependency on protobuf 2.6.1 somewhere between KCL 1.4.0 and 1.6.1.
-       -->
-    </dependency>
```

Comment:
Before SPARK-14421, it is a provided dependency. Don't we need to keep it?

Reply:
Yes, it's `provided` dependency now as I described in the PR description.
![Screenshot 2024-02-14 at 09 58 01](https://github.com/apache/spark/assets/9700541/b7e829a5-f1cc-47d4-85b4-19cedcf0e46b)

Reply:
Oh, I see. It was overriding the protobuf version.